### DISCUSSION TITLE
Publish the updates on plugin supporting code

### DISCUSSION THREAD

**Discussion by Bert-Proesmans** [FiloSottile/age#562] [General]
Hi guys, I'm writing in from the context of sops pull request https://github.com/getsops/sops/pull/1465 (read: sops, a wrapper of age, cannot currently process age keys that are handled by age plugins).

The referenced PR makes use of the age go-code as a library and specifically interacts with the exposed plugin package and abstraction structs committed here https://github.com/FiloSottile/age/compare/6976c5f...c89f0b9 (committed to master at 2023-08-05)

Do you guys think that functionality is tested comprehensively enough, and builds enough functionality scope, to publish it under a new (point-)release?

Please mention if there are blockers, or otherwise unrelated milestone information, that affect the next releasedate. I fully intend to contribute and see the above PR through!

**Comment by Bert-Proesmans** [comment]
Hi @FiloSottile, I'm pinging you in case you wanted to be aware of the question to create a new release but somehow missed this thread. Do you happen to have more details about the schedule for the next release of age?

**Comment by Bert-Proesmans** [comment]
Done with release v1.2.0 (https://github.com/FiloSottile/age/releases/tag/v1.2.0)

**Reply by FiloSottile** [reply]
Thanks for reaching out, and glad the release works for you! Feel free to ping me on SOPS issues if there's something we can do on our side to enable you.
